coralproject/talk
# Chore/corl 2456 fix high severity vulnerabilities

## What does this PR do?
This PR removes all critical and some  high-severity vulnerabilities in our NPM dependencies.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Check out this repo
2. Make sure you are using npm 3.8.0
3. `rm -rf node_modules` and `npm i`
4. `npm audit --omit=dev` and verify that there are no critical vulnerabilities, and fewer high-severity vulnerabilities
5. Run all tests
6. Build and run locally and verify that the app is working

Additional functionality that will need to be tested:
- Possibly anything involving file uploading (not sure if this is user facing functionality or just used internally by graphql)
- Rudderstack integration
- Google cloud profiling 
- Styling/style inlining of emails
-  OIDC integration
- story scraping
- cron jobs
 
## How do we deploy this PR?
No special considerations should be needed.
